docs/userguide.md

```diff
@@ -174,6 +175,13 @@ Status:
     Status:                True
     Type:                  Ready
 ```
+### Unjoin Clusters
+
+If required, federation allows you to unjoin clusters using `kubefed2` tool.
+
+```bash
+./bin/kubefed2 unjoin cluster2 --cluster-context cluster2 --host-cluster-context cluster1 --add-to-registry --v=2
```

Comment:
s/--add-to-registry/--remove-from-registry

Reply:
Sorry for that. Updated !